cardano-cli/src/Cardano/CLI/Shelley/Run/Genesis.hs

```diff
@@ -339,8 +672,8 @@ getCurrentTimePlus30 =
     plus30sec = addUTCTime (30 :: NominalDiffTime)
 
 
-readShelleyGenesis
-  :: FilePath
+readShelleyGenesis ::
+     FilePath
```

Comment:
Why the move?

Reply:
This way you can grep for the definition -- by simply adding the `::`.

Reply:
Please revert this (and other instances in this PR where you have done this). All other functions in this repo do not follow this style.

Reply:
Will green light once this is done.

Reply:
@Jimbo4350, done.